src/process_utility.c

```diff
@@ -365,8 +365,23 @@ process_copy(ProcessUtilityArgs *args)
 	Cache *hcache;
 	Oid relid;
 
+	/* We only copy for COPY FROM (which copies into a hypertable). Since
+	 * hypertable data are in the hypertable chunks and no data would be
+	 * copied, we skip the copy for COPY TO, but print an informative
+	 * message. */
 	if (!stmt->is_from || NULL == stmt->relation)
+	{
+		if (stmt->relation)
+			ereport(NOTICE,
+					(errmsg("hypertable data are in the chunks, no data will be copied"),
+					 errdetail("Data for hypertables are stored in the chunks of a hypertable so "
+							   "COPY TO of a hypertable will not copy any data."),
```

Comment:
Just a suggestion with minor rephrasing to correspond to what actually happens:
```suggestion
							   "COPY TO of a hypertable will copy no data."),
```